subprojects/core/src/main/java/org/gradle/internal/scripts/BuildScriptCompileAndInstrumentUnitOfWork.java

```diff
@@ -73,10 +81,20 @@ public Identity identify(Map<String, ValueSnapshot> identityInputs, Map<String,
         return () -> identity;
     }
 
+    @Override
+    public void visitOutputs(File workspace, OutputVisitor visitor) {
```

Comment:
:x: This should be annotated with `@OverridingMethodsMustInvokeSuper`.

Reply:
Done